features/ssh_push.feature

```diff
@@ -38,3 +43,23 @@ Feature: Register a system to be managed via SSH push
   Scenario: Check this client's contact method
     Given I am on the Systems overview page of this client
     Then I should see a "Push via SSH tunnel" text
+
+  # Salt minion bootstrap
+  Scenario: bootstrap a sles minion via ssh-push
+     Given I am authorized
+     When I follow "Salt"
+     Then I should see a "Bootstrapping" text
+     And I follow "Bootstrapping"
+     Then I should see a "Bootstrap Minions" text
+     # FIXME: we should maybe add the minion_ip in server/hosts, so we address systems without-ip this for redhat or sles.
+     And  I enter "#{$minion_ip}" as "hostname"
```

Comment:
here as mentioned, we should maybe modify the /etc/hosts of the server, so that we can put the name of the minion, and not the ip.